modules/initializeBookList.js

```diff
@@ -0,0 +1,6 @@
+import BookList from './book.js';
+
+export default function initializeBookList() {
+  const bookList = new BookList();
+  return bookList;
+}
```

Comment:
Good job in splitting up the codes into modules, however, the functionality for adding a book is currently gone and i strongly recommend your restore the functionality of the app in order to make it usable again.

[screen-capture (3).webm](https://github.com/rodrigopenayo98/Awesome-Books-Rod-ES6/assets/84930355/fe9fb246-0590-40b0-b167-10f7f7ff1a1c)